app/config/config.yml

```diff
@@ -32,6 +32,9 @@ framework:
     fragments:       ~
     http_method_override: true
     assets: ~
+    php_errors:
+        log: true
+
```

Comment:
It's from upstream https://github.com/symfony/symfony-standard/blob/master/app/config/config.yml#L34-L37

:trollface: